app/src/main/java/org/eyeseetea/malariacare/domain/usecase/GetSurveyAnsweredRatioUseCase.java

```diff
@@ -35,17 +35,15 @@ public void execute(long idSurvey, Callback callback) {
     private SurveyAnsweredRatio getSurveyWithStatusAndAnsweredRatio(long idSurvey,
```

Comment:
this use case should use AsynExecutor for retrieve from database and main executor to invoke callbacks